Makefile

```
@@ -72,6 +72,11 @@ ifeq ($(OS),Linux)
 LOCAL_LDFLAGS += -Wl,--as-needed
 LOCAL_LDLIBS += -lacl -lcap -lattr -lrt
 
+ifdef USE_ONIGURUMA
+LOCAL_LDLIBS += -lonig
+LOCAL_CFLAGS += -DUSE_ONIGURUMA
```

I try to namespace all the macros, so this should be `BFS_USE_ONIGURUMA`.  And I try to use `#if` over `#ifdef` (so that `-DBFS_SOMETHING=0` turns it off), so
```suggestion
LOCAL_CFLAGS += -DBFS_USE_ONIGURUMA=1
```